File: app/src/main/java/com/riders/thelab/data/local/model/DeviceInformation.java
package com.riders.thelab.data.local.model;


import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
public class DeviceInformation {

    private String name;
    private String brand;
    private String model;
    private String serial;
    private String fingerPrint;
    private String hardware;
    private String IMEI;
    private String id;
    private int screenWidth;
    private int screenHeight;

    private String androidVersionName;
    private int sdkVersion;
    private String androidRelease;
    private boolean rooted;


    public DeviceInformation() {
    }
}

File: app/src/main/java/com/riders/thelab/data/local/model/DeviceInformation.kt
package com.riders.thelab.data.local.model

data class DeviceInformation constructor(
        var name: String,
        var brand: String,
        var model: String,
        var serial: String,
        var fingerPrint: String,
        var hardware: String,
        var IMEI: String,
        var id: String,
        var screenWidth: Int = 0,
        var screenHeight: Int = 0,

        var androidVersionName: String,
        var sdkVersion: Int = 0,
        var androidRelease: String,
        var rooted: Boolean = false
) {
    override fun toString(): String {
        return "DeviceInformation(name=$name, brand=$brand, model=$model, serial=$serial, fingerPrint=$fingerPrint, hardware=$hardware, IMEI=$IMEI, id=$id, screenWidth=$screenWidth, screenHeight=$screenHeight, androidVersionName=$androidVersionName, sdkVersion=$sdkVersion, androidRelease=$androidRelease, rooted=$rooted)"
    }
}
